README.md

```diff
@@ -5271,69 +5335,111 @@ sleep 30
+- `rlhf.ppo.use_absolute_kl`: Whether or not to use the absolute value of the KL Divergence w.r.t. the Initial Policy.
+- `rlhf.ppo.epochs`: Number of training epochs the Actor will perform on the samples stored in the rollout buffer before generating new samples.
 - `rlhf.ppo.num_rollout_samples`: Number of samples that will be generated during the rollout stage before moving to the training stage.
-- `rlhf.ppo.rollout_micro_batch_size`: Micro batch size for the rollout phase. Each GPU will load this many prompts and generate responses for them.
-- `rlhf.ppo.ratio_eps`: epsilon value for clipping the PPO ratio during training.
-- `rlhf.ppo.discount`: discount factor for calculating the returns and advantages.
-- `rlhf.ppo.gae_lambda`: lambda value for the Generalized Advantage Estimation (GAE) calculation.
-- `rlhf.ppo.normalize_advantage`: whether to normalize the advantages to have a mean of zero and standard deviation of one.
+- `rlhf.ppo.rollout_micro_batch_size`: Micro batch size for the rollout phase. Each GPU will load this many prompts at once and generate responses for them.
+- `rlhf.ppo.ratio_eps`: Epsilon value for clipping the PPO ratio during training.
+- `rlhf.ppo.discount`: Discount factor for calculating the returns and advantages.
+- `rlhf.ppo.gae_lambda`: Lambda value for the Generalized Advantage Estimation (GAE) calculation.
+- `rlhf.ppo.normalize_advantage`: Whether or not to normalize the advantages to have a mean of zero and standard deviation of one within each global batch.
+
+Note that although the sampling parameters during the rollout phase can also be modified (through `model.sampling_params.*`), it is not recommended to do so because the implementation currently does not account for these changes when computing the log probabilities of the generated responses.
 
-During the rollout phase, the sampling parameters for the model can also be modified, by using the parameters in `model.sampling_params`.
+The Critic's config also contains a `model.rlhf` section with the following hyper-parameter:
+- `rlhf.ppo.critic_loss_clip_value`: Used in the Critic loss term that clamps the difference between the current Critic value predictions and those that were predicted during rollout generation (disabled when set to zero).
 
 #### 5.16.3. Future Work
 <a id="markdown-future-work" name="future-work"></a>
 
-- The reward model training only supports datasets with two responses per prompt. We will add support for training with datasets that have more than 2 responses per prompt in future releases.
-- The throughput of PPO will be greatly increased in future releases.
-- The stability of the PPO learning process is not good enough. We will continue working to improve the PPO learning for our models.
+- The throughput of PPO will be increased in future releases.
+- We will continue improving the stability of the PPO learning phase.
+- We will add more learning algorithms beyond PPO.
```

Comment:
* I removed the point about "The reward model training only supports datasets with two responses per prompt" because there already exists a workaround (which I described above), so I don't think it's worth highlighting it here (even if we may want to make it more convenient in the future).
* I removed "greatly" because the latest code is already much faster